front/app/containers/Admin/projects/project/permissions/components/ModeratorList.tsx

```diff
@@ -50,40 +51,50 @@ const ModeratorList = memo(
     if (!isNilOrError(authUser) && !isNilOrError(moderators)) {
       return (
         <List>
-          {moderators.map((moderator, index) => {
-            const firstName = moderator.attributes.first_name;
-            const lastName = moderator.attributes.last_name;
-            const invitationPending =
-              moderator.attributes.invite_status === 'pending';
-            const displayName = invitationPending ? (
-              <PendingInvitation>
-                {formatMessage(messages.pendingInvitation)}
-              </PendingInvitation>
-            ) : firstName && lastName ? (
-              `${firstName} ${lastName}`
-            ) : (
-              <UnknownName>{formatMessage(messages.unknownName)}</UnknownName>
-            );
+          <>
```

Comment:
Also fixed the data freshness issue here. Removed moderator didn't get removed until page refresh.